### ISSUE TITLE
failed to start chainlink container in integration test

### ISSUE THREAD

**Issue by yutongp** [smartcontractkit/external-initiator#68]
i have ran `./integration/setup` without issue.
But when i ran `./integration/run_test`, the integration_chainlink container kept restarting.
Here is the log from that container:
```
2020-09-15T00:10:35Z [INFO]  API exposed for user notreal@fakeemail.ch          cmd/local_client.go:95
2020-09-15T00:10:35Z [DEBUG] eth.Client#Dial(...)                               eth/client.go:110
error starting app: dial tcp 127.0.0.1:8546: connect: connection refused
```

i didn't change the `integration/chainlink.env` file under integration which has `ETH_DISABLED=true`

**Comment by krebernisak** [comment]
Thanks, @yutongp!

You are correct, this got broken in Chainlink node `0.8.13` release.

We are working on a fix, but until then you can fix the version to `0.8.12` in the Dockerfile to avoid the issue and make the integration tests run.

**Comment by yutongp** [comment]
@krebernisak  Thank you! Will switch to 0.8.12 for now

**Comment by krebernisak** [comment]
@yutongp should be fixed by [smartcontractkit/chainlink#3382](https://github.com/smartcontractkit/chainlink/pull/3382), which should be included in a 0.8.16 release later today.

At that point, we can switch back to `smartcontract/chainlink:latest`.

**Comment by krebernisak** [comment]
Closing this issue as 0.8.16 is now released, and I confirmed that the bug is fixed.
